docs/overview.md

```
@@ -0,0 +1,37 @@
+{% docs project_overview %}
+
+# Adventure Works Analytics — dbt Project
+
+Este projeto transforma dados brutos do SAP da **Adventure Works** em um modelo estrela
+analítico, pronto para consumo em dashboards e análises de negócio.
+
+## Arquitetura em 3 Camadas
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)